apps/patient/src/components/FAQ.tsx

```diff
@@ -37,37 +37,43 @@ const questions = [
   }
 ];
 
+export const FAQContents = () => {
+  return (
+    <Accordion allowToggle>
+      {questions.map(({ question, answer }, idx) => (
+        <AccordionItem
+          key={question}
+          borderTopWidth={idx === 0 || idx === question.length - 1 ? 0 : 1}
+          borderBottomColor="white"
+        >
+          <>
```

Comment:
unnecessary fragment

Reply:
Lol pretty sure you wrote this code.

Reply:
Oh but actually I think `AccordionItem` can only take one child

Reply:
it takes multiple